src/ingestion/process-event.ts

```diff
@@ -10,18 +10,23 @@ import { ClickHouse } from 'clickhouse'
 import { DB } from '../db'
 import { status } from '../status'
 import * as Sentry from '@sentry/node'
+import { createPosthog, DummyPostHog } from '../extensions/posthog'
 
 export class EventsProcessor {
     pluginsServer: PluginsServer
     db: DB
     clickhouse: ClickHouse
     kafkaProducer: Producer
+    posthog?: DummyPostHog
 
     constructor(pluginsServer: PluginsServer) {
         this.pluginsServer = pluginsServer
         this.db = pluginsServer.db
         this.clickhouse = pluginsServer.clickhouse!
         this.kafkaProducer = pluginsServer.kafkaProducer!
+        if (pluginsServer.POSTHOG_TEAM_ID) {
+            this.posthog = createPosthog(this.pluginsServer, parseInt(pluginsServer.POSTHOG_TEAM_ID))
+        }
```

Comment:
In python we're emitting this "first team event ingested" event to app.posthog.com. The code above would do that only if co-located with posthog in the same database (so, cloud only). We should be able to capture these events no matter where this is running.

I'd swap this out with `posthog-js-lite` or even a raw API call. `-js-lite` is a pretty bad library, but it's good enough to abstract capture calls into fetch requests for the toolbar, so it could also work here.

Reply:
Totally forgot about instances other than Cloud! `posthog-js-lite` will do the job just fine I think, rewritten to use it